spec/std/file_spec.cr

```diff
@@ -124,6 +124,10 @@ describe "File" do
     File.basename("").should eq("")
   end
 
+  it "gets basename when / (slash)" do
+      File.basename("/").should eq("")
```

Comment:
Please use 2 spaces here. Alternative you can do `crystal tool format [filename]` to fix this.